roc_example_test.go

```diff
@@ -77,7 +79,8 @@ func ExampleROC_equallySpacedCutoffs() {
 	n := 9
 
 	stat.SortWeightedLabeled(y, classes, weights)
-	cutoffs := stat.EquallySpaced(y[0], y[len(y)-1], n)
+	cutoffs := make([]float64, n)
+	floats.Span(cutoffs, math.Nextafter(y[0], y[0]-1), y[len(y)-1])
```

Comment:
We have the same problem in stat.Histogram. We should make sure that these two functions have the problem in the same direction, or resolve the problem with the same hack.